pkg/network/ebpf/c/prebuilt/tracer.c

```diff
@@ -280,34 +280,41 @@ static __always_inline int read_conn_tuple(conn_tuple_t* t, struct sock* skp, u6
     return read_conn_tuple_partial(t, skp, pid_tgid, type);
 }
 
-static __always_inline void handle_tcp_stats(conn_tuple_t* t, struct sock* sk) {
+static __always_inline void handle_tcp_stats(conn_tuple_t* t, struct sock* sk, __u32* segs_in, __u32* segs_out) {
     u32 rtt = 0;
     u32 rtt_var = 0;
     bpf_probe_read(&rtt, sizeof(rtt), ((char*)sk) + offset_rtt());
     bpf_probe_read(&rtt_var, sizeof(rtt_var), ((char*)sk) + offset_rtt_var());
 
+    segs_in = 0;
```

Comment:
This sets the pointer to `0`, rather than the value. Given my other suggestion, I think you can remove this code altogether.